configs/tsimx6_ubuntu_24_04_headless_defconfig

```diff
@@ -0,0 +1,34 @@
+CONFIG_DS_DISTRO_UBUNTU_24_04=y
+CONFIG_DS_PACKAGELIST="ubuntu_24.04_headless.txt"
+CONFIG_DS_KERNEL_PROVIDER_GIT_URL="https://github.com/embeddedTS/linux-lts.git"
+CONFIG_DS_KERNEL_PROVIDER_GIT_VERSION="v6.6.47.1-ts"
+CONFIG_DS_KERNEL_DEFCONFIG="tsimx6_defconfig"
+CONFIG_DS_KERNEL_INSTALL_DEVICETREE_FILESYSTEM="imx6dl-ts4900-14 imx6dl-ts4900-2 imx6dl-ts4900 imx6dl-ts7970 imx6dl-ts7990-lxd imx6dl-ts7990-lxd-revb imx6dl-ts7990-microtips imx6dl-ts7990-microtips-revb imx6dl-ts7990-okaya imx6dl-ts7990-okaya-revb imx6q-ts4900-14 imx6q-ts4900-2 imx6q-ts4900 imx6q-ts7970 imx6q-ts7990-lxd imx6q-ts7990-lxd-revb imx6q-ts7990-microtips imx6q-ts7990-microtips-revb imx6q-ts7990-okaya imx6q-ts7990-okaya-revb"
+CONFIG_DS_KERNEL_INSTALL_ZIMAGE_FILESYSTEM=y
+CONFIG_DS_KERNEL_INSTALL_UIMAGE_FILESYSTEM=y
+CONFIG_DS_KERNEL_INSTALL_UIMAGE_LOADADDR=0x10008000
+CONFIG_DS_COMPONENT_LEGACY_BOOTSCRIPT=y
+CONFIG_DS_COMPONENT_LEGACY_BOOTSCRIPT_CMDLINE="console=ttymxc0,115200 init=/sbin/init loglevel=4"
+CONFIG_DS_COMPONENT_LEGACY_BOOTSCRIPT_INSTALL_SOURCE=y
+CONFIG_DS_COMPONENT_TS4900_UTILS=y
+CONFIG_DS_COMPONENT_TS4900_FPGA=y
+CONFIG_DS_COMPONENT_TSSUPERVISORUPDATE=y
+CONFIG_DS_COMPONENT_TS_UBOOT_CONFIGS=y
+CONFIG_DS_COMPONENT_XORG_ETNAVIV=y
+CONFIG_DS_COMPONENT_WILC3000_FIRMWARE=y
+CONFIG_DS_MODULE_WILC3000=y
+CONFIG_DS_MODULE_WILC3000_GIT_VERSION="linux4microchip-2024.04-community"
+CONFIG_DS_COMPONENT_WL12XX_FIRMWARE=y
+CONFIG_DS_COMPONENT_LINUX_FIRMWARE=y
+CONFIG_DS_COMPONENT_LINUX_FIRMWARE_QCA9377=y
+CONFIG_DS_WATCHDOG=y
+CONFIG_DS_COMPONENT_RESIZE=y
+CONFIG_DS_USER=y
+CONFIG_DS_USER_PASSWORD="user"
+CONFIG_DS_USER_SUDO=y
+CONFIG_DS_PURGE_LOCALES=y
+CONFIG_DS_HOSTNAME="tsimx6"
+CONFIG_DS_TIMEZONE="America/Phoenix"
+CONFIG_DS_DOWNGRADE_WPA_HOSTAPD=y
```

Comment:
This may not be necessary with 6.6. Rather than holding this up, can you open an Issue to remind us to test 6.6 with wpa_supplicant 2.10?

Reply:
Good point, I'll open that issue.